### PR TITLE
multitail: disable due to repo got removed

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by gromgit** [comment]
Now available via `brew install gromgit/new-life/multitail`. :grinning:

File: Formula/multitail.rb [hunk 1]
```diff
@@ -14,6 +14,8 @@ class Multitail < Formula
     sha256 cellar: :any, high_sierra:   "57526de43035b0d5d2520d54b252d29c20a4efb146c019ac044ad5067be5351a"
   end
 
+  disable! date: "202-05-17", because: :repo_removed
```
[the remark this answers is not in the document]
Comment (SMillerDev) [reply Formula/multitail.rb:17]: Not 2021?

[the remark this answers is not in the document]
Comment (iMichka) [reply Formula/multitail.rb:17]: Lol yes, 2021 :D

[the remark this answers is not in the document]
Comment (carlocab) [reply Formula/multitail.rb:17]: Can we disable this for a future date? The bottles still work, after all.

**Comment by carlocab** [comment]
Deprecated in #77690, which is probably good enough for now.

File: Formula/multitail.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (chenrui333) [reply Formula/multitail.rb]: my bad 🤦

**Comment by chenrui333** [comment]
> Deprecated in #77690, which is probably good enough for now.

Sounds great, closing this now.

**Pull request closed without merging**